bumble/gatt_server.py

```diff
@@ -222,7 +222,13 @@ def add_service(self, service: Service):
         # Add the service attribute to the DB
         self.add_attribute(service)
 
-        # TODO: add included services
+        # Add all included service
+        for included_service in service.included_services:
+            # Not registered yet, register the included service first.
+            if not included_service.registered:
```

Comment:
see earlier comment about keeping that state in the Server class.

Reply:
Done